.github/workflows/release.yml

```diff
@@ -20,22 +20,23 @@ jobs:
       contents: write
 
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/download-artifact@v3
+      - uses: actions/checkout@v4
+      - uses: actions/download-artifact@v4
         with:
-          name: libfuzzer-dotnet
+          pattern: libfuzzer-dotnet-*
+          merge-multiple: true
```

Comment:
Due to https://github.com/actions/download-artifact/blob/main/docs/MIGRATION.md#multiple-uploads-to-the-same-named-artifact